sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/batch_deployment.py

```diff
@@ -32,7 +33,7 @@
 
 
 class BatchDeployment(Deployment):  # pylint: disable=too-many-instance-attributes
-    """Batch endpoint deployment entity.
+    """Deprecated. Batch endpoint deployment entity.
```

Comment:
We should add the comment here that directs customers to the correct `ModelBatchDeployment` object.